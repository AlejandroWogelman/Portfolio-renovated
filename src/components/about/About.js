import React from "react";
import { viewModal } from "../Modal";
import "./about.css";

export const About = () => {
  return (
    <>
      <section className="about-container" id="about">
        <article>
          <h2>about me</h2>
          <p className="mt">
            Hola! Soy Alejandro Wogelman. Desde hace un tiempo me encuentro
            aprendiendo y aplicando cada nuevo concepto.
            <br />
            <br />
            Actualmente me encuentro en la busqueda para poder integrar un
            equipo de trabajo que me permita seguir aprendiendo, mejorando y
            aplicando.
          </p>
          <br /> <p>Para mejorar hay que HACER!</p>
          <br />
          <hr />
          <div className="about-img-hob">
            <div>
              <ul>
                <li>
                  Gym:
                  <img
                    src={
                      require("../../assets/Fitness_Monochromatic.svg").default
                    }
                    alt="gym"
                    width="80px"
                  />
                </li>
                <li>
                  Futbol:
                  <img
                    src={require("../../assets/Soccer_Isometric.svg").default}
                    alt="soccer"
                    width="80px"
                  />
                </li>

                <li>
                  Cursos:
                  <img
                    src={
                      require("../../assets/Javascript_Isometric.svg").default
                    }
                    alt="cursos"
                    width="80px"
                  />
                </li>
                <li>
                  Anime:
                  <img
                    src={
                      require("../../assets/ball_dragon_game_movie_icon_183186.svg")
                        .default
                    }
                    alt="anime"
                    width="50px"
                  />
                </li>
              </ul>
            </div>
            <div className="img-about-container">
              <img
                className="about-img"
                src={require("../../assets/perfilImageShort.jpg").default}
                alt="perfil"
                width="180px"
                height="180px"
              />
              <button className="btn-cv" onClick={viewModal}>
                CV
              </button>
            </div>
          </div>
        </article>
      </section>
      <article className="skills-container">
        <div>
          <h3>Skills</h3>
          <p style={{ margin: "0px", fontSize: "1.5rem" }}>Experiencia en</p>
        </div>
        <div className="skills-content">
          <p>HTML</p>
          <p>REACT</p>
          <p>CSS</p>
          <p>TYPESCRIPT</p>
          <p>JAVASCRIPT</p>
        </div>
      </article>
      <article className="metodologias-container">
        <div>
          <h3>Metodolog√≠a</h3>
          <p style={{ margin: "0px", fontSize: "1.5rem" }}>SCRUM</p>
        </div>
        <div className="metodologias-content">
          <p>BITBUCKET</p>
          <p>JIRA</p>
          <p>CLOCKIFY</p>
        </div>
      </article>
    </>
  );
};
