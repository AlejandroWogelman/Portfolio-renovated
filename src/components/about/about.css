.about-container {
  color: rgb(140, 140, 140);
  padding: 7% 15% 5% 15%;
  text-align: center;
}
.about-container article h2 {
  font-size: 2.3rem;
}
.mt {
  font-family: "Comfortaa", cursive;
  font-size: 1.5rem;
  margin-top: 3rem;
  padding: 5px 0;
}
.about-img-hob {
  display: flex;
  justify-content: center;
  margin-bottom: 2.5rem;
}
.about-img-hob ul {
  display: flex;
  width: auto;
  flex-wrap: wrap;
}
.about-img-hob ul li {
  display: flex;
  width: 25%;
  justify-content: center;
  align-items: center;
}
.about-img {
  margin-top: 2rem;
  border-radius: 5px;
}
.img-about-container {
  width: 50%;
}
.skills-container,
.metodologias-container {
  display: flex;
  justify-content: space-evenly;
  align-items: center;
  background-color: rgb(244, 189, 51);
  color: rgb(247, 242, 232);
  padding: 5%;
  font-size: 1.8rem;
}
.metodologias-container {
  background-color: rgb(79, 197, 144);
}
.skills-content,
.metodologias-content {
  display: flex;
}
.skills-content p,
.metodologias-content p {
  font-weight: bold;
  font-size: 1.3rem;
  margin: 0 1rem;
}

@media screen and (max-width: 720px) {
  .about-container {
    display: flex;
    justify-content: center;
  }
  .about-img-hob {
    flex-direction: column;
    align-items: center;
  }
  .img-about-container {
    width: 100%;
  }
  .skills-container,
  .metodologias-container {
    flex-direction: column;
    text-align: center;
  }
  .skills-content p,
  .metodologias-content p {
    font-size: 1.1rem;
    margin-top: 1.3rem;
  }
}
@media screen and (max-width: 500px) {
  .skills-container,
  .metodologias-container {
    padding: 15%;
  }
}
