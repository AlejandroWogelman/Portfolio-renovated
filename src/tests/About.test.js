import { About } from "../components/about/About";
import { shallow } from "enzyme";
import "@testing-library/jest-dom";

describe("<About/>", () => {
  const wrapper = shallow(<About />);

  test("rendered of contact component", () => {
    expect(wrapper).toMatchSnapshot();
  });
  test("View description", () => {
    const descrip = wrapper.find("p").at(0).html();
    expect(descrip.includes("Alejandro Wogelman")).toBe(true);
  });

  test("imgs correct path", () => {
    const div = wrapper.find(".about-img-hob").find("li");

    const gym = div.at(0).find("img");
    const futbol = div.at(1).find("img");
    const cursos = div.at(2).find("img");
    const anime = div.at(3).find("img");

    expect(futbol.prop("src")).toBe("Soccer_Isometric.svg");
    expect(anime.prop("src")).toBe("ball_dragon_game_movie_icon_183186.svg");
    expect(cursos.prop("src")).toBe("Javascript_Isometric.svg");
    expect(gym.prop("src")).toBe("Fitness_Monochromatic.svg");
  });
});
