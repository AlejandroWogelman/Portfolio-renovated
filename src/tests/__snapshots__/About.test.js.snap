// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About/> rendered of contact component 1`] = `
<Fragment>
  <section
    className="about-container"
    id="about"
  >
    <article>
      <h2>
        about me
      </h2>
      <p
        className="mt"
      >
        Hola! Soy Alejandro Wogelman. Desde hace un año me encuentro aprendiendo y aplicando cada nuevo concepto. Actualmente me encuentro en la busqueda para poder integrar equipos de trabajo ya que creo no hay mejor forma de mejorar que HACER!
      </p>
      <br />
      <hr />
      <div
        className="about-img-hob"
      >
        <div>
          <ul>
            <li>
              Gym:
              <img
                alt="gym"
                src="Fitness_Monochromatic.svg"
                width="80px"
              />
            </li>
            <li>
              Futbol:
              <img
                alt="soccer"
                src="Soccer_Isometric.svg"
                width="80px"
              />
            </li>
            <li>
              Cursos:
              <img
                alt="cursos"
                src="Javascript_Isometric.svg"
                width="80px"
              />
            </li>
            <li>
              Anime:
              <img
                alt="anime"
                src="ball_dragon_game_movie_icon_183186.svg"
                width="50px"
              />
            </li>
          </ul>
        </div>
        <div
          className="img-about-container"
        >
          <img
            alt="perfil"
            className="about-img"
            height="180px"
            width="180px"
          />
          <a
            href="https://drive.google.com/file/d/11bHKZeBW3yv8iY4AfQpvwcBDTu17LfGq/view?usp=sharing"
            rel="noreferrer"
            target="_blank"
          >
            CV
          </a>
        </div>
      </div>
    </article>
  </section>
  <article
    className="skills-container"
  >
    <div>
      <h3>
        Skills
      </h3>
      <p
        style={
          Object {
            "fontSize": "1.5rem",
            "margin": "0px",
          }
        }
      >
        Experiencia en
      </p>
    </div>
    <div
      className="skills-content"
    >
      <p>
        HTML
      </p>
      <p>
        CSS
      </p>
      <p>
        JAVASCRIPT
      </p>
      <p>
        REACT
      </p>
    </div>
  </article>
  <article
    className="metodologias-container"
  >
    <div>
      <h3>
        Metodología
      </h3>
      <p
        style={
          Object {
            "fontSize": "1.5rem",
            "margin": "0px",
          }
        }
      >
        SCRUM
      </p>
    </div>
    <div
      className="metodologias-content"
    >
      <p>
        BITBUCKET
      </p>
      <p>
        JIRA
      </p>
      <p>
        CLOCKIFY
      </p>
    </div>
  </article>
</Fragment>
`;
